### ISSUE TITLE
Add Reset Button to all the functions.

### ISSUE THREAD

**Issue by amajaying** [developer-diganta/Dino#51]
### Subject of the issue
Describe your issue here.

### Your environment
* version of angular-translate
* version of angular
* which browser and its version

### Steps to reproduce
Tell us how to reproduce this issue. Please provide a working demo, you can use [this template](https://plnkr.co/edit/XorWgI?p=preview) as a base.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

**Comment by developer-diganta** [comment]
Yeah this would be helpful. I guess for some functions some are already working for the revert button. Just check and make sure you do the other ones.

**Comment by Mridul-2003** [comment]
Plz assign me this issue

**Comment by developer-diganta** [comment]
@amajaying are you working on this issue?

**Comment by amajaying** [comment]
I am so sorry, I have been caught up with some uni assignments and assessments. It is okay if @Mridul-2003 wants to work on this. You can assign him the issue.
